sandy/sections/mf2.py

```diff
@@ -0,0 +1,367 @@
+"""
+This module contains only two public functions:
+
+    * `read_mf2`
+    * `write_mf8`
+
+Function `read` reads a MF2/MT section from a string and produces a content
+object with a dictionary-like structure.
+The content object can be accessed using most of the keywords specified in
+the ENDF6 manual for this specific MF section.
+
+Function `write_mf2` writes a content object for a MF2/MT section into a
+string.
+MAT, MF, MT and line numbers are also added (each line ends with a `\n`).
+"""
+
+import sandy
+
+import numpy as np 
+
+__author__ = "Rayan HADDAD"
+__all__ = [
+        "read_mf2",
+        ]
+
+
+mf = 2
+mt = 151
+
+def read_mf2(tape, mat):
+        
```

Comment:
docstring with a example, inputs and the outputs helps users to know how to use the function.